doc/api/mediaCapabilitiesProber/browserAPI.md

```diff
@@ -0,0 +1,22 @@
+### Probed APIS
```

Comment:
I'm not sure the mediaCapabilitiesProber needs to be its own directory. Just a single page should be enough